src/Components/ItemsList.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import UserInput from "./UserInput";
+import Items from "./Items";
+import { Button } from "react-bootstrap";
+
+export default class ItemsList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      itemsList: [],
+    };
+  }
+  //We need to be able to add items to the list
+  //Items are added in the UserInput component: when user inputs something into the input field and then clicks "add item" , the item should get added into the itemsArray
+  //Each item can be deleted
+  //Each item can be changed
+  //Each item can be edited
+  //Each item has checkbox, once the user got the item, they may tick the checkbox and the item will get crossed out
+
+  //1) adding items logic
+  //this function is passed as props to UserInput
+  addItem = (inputValue) => {
+    // const currentDate = new Date().toDateString();
+    // console.log(currentDate);
+    let newItem = { name: " ", key: " ", isChecked: false };
+    newItem.name = inputValue;
+    newItem.key = Date.now();
```

Comment:
```suggestion
    const newItem = { name: inputValue, key: Date.now(), isChecked: false };
```

Objects and arrays should always be `const`. If we set keys of that object to another value anyways right after definition, why not just define it that way either way?